storage/innobase/xtrabackup/test/suites/pagetracking/encryption_after_checkpoint.sh

```diff
@@ -0,0 +1,61 @@
+############################################################################
+#PXB-2681 PXB crashes with alter encryption='n' happens after the checkpoint
+############################################################################
+
+. inc/common.sh
+. inc/keyring_file.sh
+
+require_debug_server
+
+start_server
+
+vlog "take incremental backup"
+mysql test <<EOF
+CREATE TABLE t1 (i serial) KEY_BLOCK_SIZE=2 ENCRYPTION='Y';
+CREATE TABLESPACE tab02k_e ADD DATAFILE 'tab02k_e.ibd'  FILE_BLOCK_SIZE 02k ENCRYPTION='Y';
+CREATE TABLE t2 (i serial) KEY_BLOCK_SIZE=2 TABLESPACE tab02k_e ENCRYPTION='Y';
```

Comment:
Is this bug only reproducible with KEY_BLOCK_SIZE set?

Reply:
No, corruption is seen without KEY_BLOCK_SIZE, but with KEY_BLOCK_SIZE is easily crashes also.